src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -940,7 +940,7 @@ The storage type of ``exp`` output is always dense
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseOut{"_mul"});
 
 // log
-MXNET_OPERATOR_REGISTER_UNARY_WITH_SPARSE_DR(log, cpu, mshadow_op::log)
+MXNET_OPERATOR_REGISTER_UNARY(log)
```

Comment:
Sorry, I do not understand this change. Can you please explain?

Reply:
the defination of  `MXNET_OPERATOR_REGISTER_UNARY_WITH_SPARSE_DR`  in `elemwise_unary_op.h` is here:
```c++
#define MXNET_OPERATOR_REGISTER_UNARY_WITH_SPARSE_DR(__name$, __xpu$, __kernel$)        \
  MXNET_OPERATOR_REGISTER_UNARY(__name$)                                                \
  .set_attr<FCompute>("FCompute<" #__xpu$ ">", UnaryOp::Compute<__xpu$, __kernel$>)
```
it will set FCompute attribute to `UnaryOp::Compute` automatically. To call Our `LogCompute`, we replace it with below codes.
```c++
MXNET_OPERATOR_REGISTER_UNARY(log)
.set_attr<FCompute>("FCompute<cpu>", UnaryOp::LogCompute<cpu, mshadow_op::log>)
```